### DISCUSSION TITLE
USB Adapter not active after Big Sur installation

### DISCUSSION THREAD

**Discussion by Augeydoggy** [chris1111/Wireless-USB-Big-Sur-Adapter#289] [Q&A]
I installed the Big Sur drivers on my 2015 Macbook Pro running Monterey 12.6. I disabled the SIP, gave the necessary permissions, and it said it had installed. I am using a Netgear A6150, which now shows up in the network preferences, but as not connected. I am not a programmer, but all the steps seemed to have gone correctly, so don't know what to do next.

**Comment by chris1111** [comment]
Did you allow software update in Security & Privacy

![Screenshot 2023-09-20 at 20 05 03](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/2bcf8285-54b9-451e-9aeb-e3829e63677a)

**Comment by Augeydoggy** [comment]
Yes, I remember doing that.

**Reply by chris1111** [reply]
If you doing that the driver is load correctly so try diferant USB port because this adapter is suppose to work it is marked as known working.

**Comment by Augeydoggy** [comment]
Thanks, hope I am not wasting your time. Here is the Security & Privacy:

[image: Screen Shot 2023-09-21 at 11.43.13 AM.png]

When I insert the 6150, it shows in Network Preferences like this:

[image: Screen Shot 2023-09-21 at 11.38.22 AM.png]
Then the PBC starts and just repeats every 5 seconds or so. I don't know if
I am supposed to be doing something when that happens.

[image: Screen Shot 2023-09-21 at 11.41.34 AM.png]

**Comment by chris1111** [comment]
In any case I should have told you from the start that you opened a discussion but that's an issue
[Read the rule](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/33)
so I transferred [here](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/issues/291)
